applications/prompt-proto-service-lsstcomcam/values.yaml

```diff
@@ -76,10 +76,6 @@ prompt-proto-service:
     # -- Set this to disable validation of S3 bucket names, allowing Ceph multi-tenant colon-separated names to be used.
     disableBucketValidation: 0
 
-  # -- The URI to a microservice that maps image metadata to a file location.
-  # If empty, Prompt Processing does not use a microservice.
-  raw_microservice: ""
-
```

Comment:
Any reason to delete this?

Reply:
Nope, that was an accident. Thanks for catching that!